src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -584,14 +584,6 @@ static inline int MPIDI_NM_mpi_put(const void *origin_addr,
         goto fn_exit;
     }
 
-    MPIDI_CH4U_EPOCH_CHECK_SYNC(win, mpi_errno, goto fn_fail);
```

Comment:
Now after looking at CH3, maybe these should just stay where they are? It looks like we first check to make sure the window is accessible, and then check for `MPI_PROC_NULL` in `ch3u_rma_ops.c`.